### DISCUSSION TITLE
User Groups - Manually add groups to manage content across instances

### DISCUSSION THREAD

**Discussion by ZarTrox** [Azure/api-management-developer-portal#2603] [Feature request]
## Description

We maintain and test our DevPortal pages centrally in one source and two test APIM instances to which only the contributors/active users should have access. In this test instance not all APIs are available as in the different target APIM instances.
We deploy the backup of our source instance to several other instances for different purposes and with different permissions (e.g. formal test, sandbox, production). Each of these APIM instances are available for different user groups and contains different APIs and different API version/revision states.

Also, if we assign certain permissions/user groups to certain content in the DevPortal content source, these user groups should only be able to see the content if they are authorized to access the DevPortal pages for the Developer Portal of the respective APIM target instance (e.g. Sandbox). 

## Feature Request

We need the ability to assign certain content to user groups that do not have access to the source APIM instance, but should be able to see certain content in certain target instances. Currently, user groups can only be selected from a predefined list. We need the possibility to enter new group names from our AAD for our tenant via the search field and add them to the list. These group names should be specially marked as additionally added. Afterwards it should be possible. To be able to select both existing and subsequently added user group names and assign them to specific page content.

## Is your portal managed or self-hosted?

Managed

**Comment by ghost** [comment]
@ZarTrox, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@ZarTrox, thank you for requesting this feature.
